api/gen_server.go

```diff
@@ -31,9 +31,6 @@ type ServerInterface interface {
 	// Get Clinic by Share Code
 	// (GET /v1/clinics/share_code/{shareCode})
 	GetClinicByShareCode(ctx echo.Context, shareCode string) error
-
-	// (DELETE /v1/clinics/{clinicId})
-	DeleteClinic(ctx echo.Context, clinicId ClinicId) error
```

Comment:
This shouldn't be deleted

Reply:
Good spot, it was on the todo list, i did not notice when DeleteClinic got merged.